docs/src/dedicated-gen-2/overview.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Platform.sh Dedicated Generation 2"
+weight: 1
+sidebarTitle: "Overview"
+layout: single
+---
+
+Platform.sh Dedicated Generation 2 provides a scalable solution as an additional option on top of your existing Grid applications.
+It provides redundant configuration with a minimum of three Virtual Machine instances. Every service is replicated across all three virtual machines in a failover configuration (as opposed to sharding), allowing a site to remain up even if one of the VMs is lost entirely.
+Dedicated Generation 2 works nearly identically to Grid environments, and does not require additional configuration on your part. The only difference is that of service availability.
+
+## Why a Generation 2?
+
+Even though our existing [dedicated infrastructure](/dedicated/overview) ensures better uptime and availability for your applications and services, configuration or application topology changes have to go through our Customer Success team, as the VMs are provisioned by us. Generation 2 gives you both the high availability from our Dedicated offer and the self-service flexibility and features of the Grid. This means that you will now be able to edit your configuration yourself – as you would do on the Grid in your `.platform/services.yaml` and `.platform.app.yaml` files – and see those changes in your Dedicated Generation 2 environments when pushed without having to open a ticket.
```

Comment:
Start first sentence with "Our existing" and perhaps change "better" to "increased," so as to not undersell grid...